DOCSP-44993 Updates for Second Draft: Backups Page (#94)
mongodb/docs-atlas-architecture: source/backups.txt

  the cloud provider's underlying backup snapshot mechanism for low cost
  and fast restores. You choose a backup policy that specifies a certain
  number of daily, weekly, and monthly snapshots.
- Continuous Cloud Backups: This is an additive feature to cloud backups

>   that provides Point In Time (PIT) recovery. This allows you to recover

Reviewer: ```suggestion
  that provides Point In Time (PIT) recovery. This feature allows you to recover
```

  back to a specific minute during the restore process by backing up the

>   oplog and capturing data changes between snapshots. This allows you to

Reviewer: ```suggestion
  oplog and capturing data changes between snapshots. This feature allows you to
```

Features
~~~~~~~~

|service| provides fully-managed backups of your data, including
point-in-time data recovery and consistent, cluster-wide snapshots of
all {+clusters+}, including sharded {+clusters+}. In |service|, you can
choose from four snapshot frequencies: hourly, daily, weekly, and

> monthly, ach with its own retention period.

Reviewer: **Typo:**

```suggestion
monthly, each with its own retention period.
```

.. list-table:: 
   :widths: 20 80 
   :stub-columns: 1

   * - Cloud Backups

>      - This provides localized backup storage using the native snapshot
>        functionality of your {+cluster+}'s cloud service provider.
>        Benefits include a strong default backup retention schedule of 12
>        months, full flexibility to customize snapshot and retention
>        schedules, and the ability to set different snapshot frequencies
>        (such as hourly for recovery, weekly or monthly for long-term
>        retention) to meet industry regulations. You can access your
>        backup data instantly, which is useful for auditing, compliance,
>        or data recovery purposes and also run queries directly against
>        the backup data, saving time and resources.  
> 
>    * - Continuous Cloud Backups
>      - This provides Point In Time (PIT) recovery, which allows you to
>        recover back to any timestamp. This allows you to recover your
>        data to the exact moment (a point in time) right before any
>        failure or event, like a cyber attack. You can also set a
>        customized restore window to dictate how many days you  would
>        like to be able to restore back to a specific point in time.  
> 
>    * - Multi-region Snapshot Distribution
>      - This allows you to increase resilience by automatically
>        distributing backup snapshots and oplogs across geographic
>        regions instead of just storing them in their primary region. You
>        can meet compliance requirements of storing backups in different,
>        air gapped geographical locations to ensure disaster recovery in
>        case of regional outages. 

Reviewer: ```suggestion
     - This feature provides localized backup storage using the native snapshot
       functionality of your {+cluster+}'s cloud service provider.
       Benefits include a strong default backup retention schedule of 12
       months, full flexibility to customize snapshot and retention
       schedules, and the ability to set different snapshot frequencies
       (such as hourly for recovery, weekly or monthly for long-term
       retention) to meet industry regulations. You can access your
       backup data instantly, which is useful for auditing, compliance,
       or data recovery purposes and also run queries directly against
       the backup data, saving time and resources.  

   * - Continuous Cloud Backups
     - This feature provides Point In Time (PIT) recovery, which allows you to
       recover back to any timestamp. This allows you to recover your
       data to the exact moment (a point in time) right before any
       failure or event, like a cyber attack. You can also set a
       customized restore window to dictate how many days you  would
       like to be able to restore back to a specific point in time.  

   * - Multi-region Snapshot Distribution
     - This feature allows you to increase resilience by automatically
       distributing backup snapshots and oplogs across geographic
       regions instead of just storing them in their primary region. You
       can meet compliance requirements of storing backups in different,
       air gapped geographical locations to ensure disaster recovery in
       case of regional outages. 
```

       To learn more, see :atlas:`Snapshot Distribution </backup/cloud-backup/snapshot-distribution/>`.

   * - Backup Compliance Policy 
     - This feature enables you to further secure business critical data
       by preventing all snapshots and oplogs stored in |service| from
       being modified or deleted for a predefined retention period
       specified by you, guaranteeing that your backups are fully WORM
       (Write Once Read Many) compliant. Only a designated, authorized
       user can turn off this protection after completing a verification

>        process with MongoDB support. This adds a mandatory manual delay

Reviewer: ```suggestion
       process with MongoDB support. This feature adds a mandatory manual delay
```

external disaster recovery region, ensuring data recovery even during
regional outages. For an |service| {+cluster+} in three regions,
multi-region Snapshot Distribution copies backups to two secondary
regions, enabling restores by using backup copies. You can also copy
critical backups, with the point-in-time data, to any secondary region
available from your cloud provider in |service| .

> When configuring your snapshot frequency, retention, and distribution,

Reviewer: ```suggestion
When you configure your snapshot frequency, retention, and distribution,
```

Recommendations for PIT Recovery
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Continuous Cloud Backups enable precise Point In Time (PIT) recovery, minimizing

Reviewer: ```suggestion
Continuous Cloud Backups enable precise Point In Time (PIT) recovery, which minimizes
```

data loss during failures. |service| can quickly recover to the exact
timestamp before a failure event, giving you at least a one minute
:abbr:`RPO (Recovery Point Objective)` and an :abbr:`RTO (Recovery Time
Objective)` of less than 15 minutes when utilizing optimized restores,
even in the event of the outage of the primary region. This is because
|service| restores the most recent snapshot from before the desired
point in time and then replays the oplog changes to restore to that
particular point. Recovery times can vary due to cloud provider disk

> warming and how much of the oplog must be replayed during recovery. Your

Reviewer: ```suggestion
warming and how much of the oplog must replay during recovery. Your
```
